go.mod

```diff
@@ -1,8 +1,9 @@
 module github.com/equinix/terraform-provider-equinix
 
-go 1.17
+go 1.18
```

Comment:
If #219 lands first, this change will need to be synced with the upstream.

Reply:
updated